src/config/dynamic-theme-fixes.config

```diff
@@ -27977,123 +27977,88 @@ CSS
 
 ================================
 
-wikibooks.org
-*.wikibooks.org
-
-INVERT
-.mwe-math-fallback-image-inline
-.mwe-math-fallback-image-display
-.bookend
-img[alt="audio speaker icon"]
-
-IGNORE INLINE STYLE
-.infobox td
-
-================================
-
-wikidata.org
-*.wikidata.org
-
-INVERT
-.mwe-math-fallback-image-inline
-.mwe-math-fallback-image-display
-.wd-mp-headerimage
-img[alt="audio speaker icon"]
-
-IGNORE IMAGE ANALYSIS
-.mw-wiki-logo
-
-================================
-
 wikimapia.org
 
 INVERT
 #map
 
 ================================
 
-wikimedia.de
-
-INVERT
-.logo
-
-================================
-
-wikimedia.org
-*.wikimedia.org
-
-INVERT
-img[src="images/black.png"]
-img[alt="audio speaker icon"]
-
-CSS
-div.mw-warning-with-logexcerpt,
-div.mw-lag-warn-high,
-div.mw-cascadeprotectedwarning,
-div#mw-protect-cascadeon,
-div.titleblacklist-warning,
-div.locked-warning {
-    background-color: ${#ffa9a9} !important;
-    border-color: ${#ffa1a1} !important;
-}
-
-================================
-
-wikimediafoundation.org
-
-INVERT
-.nav-logo
-
-================================
-
 wikimediastatus.net
+*.wikimediastatus.net
 
 INVERT
+.logo
 .logo-container
 
 ================================
 
+wikibooks.org
```

Comment:
As you added wikibooks.org as the first one of the sites with fix then the whole fix for this sites should be before wikimapia.org

Now the tests ends with error because the sites are not ordered alphabeticaly.